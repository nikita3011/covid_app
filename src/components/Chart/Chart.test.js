import React from 'react';
import { shallow, configure } from 'enzyme';
import { findByTestAtrr } from "../../../utils/index";

import Adapter from "enzyme-adapter-react-16";
import Chart from './Chart';

configure({ adapter: new Adapter() });

// describe('MyComponent', () => {
//   it('should render correctly in "debug" mode', () => {
//     const component = shallow(<Chart/>);
  
//     expect(component).toMatchSnapshot();
//   });
// });


describe("Chart", () => {
    describe("chart test", () => {
      let wrapper;
      beforeEach(() => {
        const props = {
            data:{
                confirmed:1,
                recovered:2,
                deaths:3,
                lastUpdate:4
            }
        };
        wrapper = shallow(<Chart {...props} />);
      });
  
      it("should render chart", () => {
        const cardComponent = findByTestAtrr(wrapper, "chart");
        // expect(cardComponent.length).toBe(3);
        expect(cardComponent).toMatchSnapshot();
      });
    });
  });