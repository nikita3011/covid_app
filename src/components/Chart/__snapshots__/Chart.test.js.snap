// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chart chart test should render chart 1`] = `ShallowWrapper {}`;
